### PR TITLE
batch of fixes for the tutorial

### PR DESCRIPTION
**Pull request by skmd0**
Mostly gramatical and brevity/phrasing fixes for the tutorial. I have made changes until the undo section.

@VitalyAnkh @DogLooksGood @Linerre Please feel free to comment on any changes I have made. I will make additional modifications if necessary.

### CODE REVIEW

File: meow-tutor.el [hunk 1]
```diff
@@ -166,61 +165,58 @@
...
+ serves as a fast way to move around the text. Below are listed some
+ other useful motions.
 
  Some common motions include:
-   \\[meow-next-word] - Move forward to before the beginning of the next word.
-   \\[meow-back-word] - Move backward to the beginning of the closest backward word.
-   \\[meow-next-symbol] - Select the next symbol, symbol means all the
+   \\[meow-next-word] - Moves the cursor forward to the end of the current word.
+   \\[meow-back-word] - Moves the cursor backward to the beginning of the current word.
+   \\[meow-next-symbol] - Moves the cursor to the start of the current symbol.
```
Comment (skmd0) [inline meow-tutor.el:197]: Oops, looks like here should be `end` instead of `start`. Will fix it after other people review.

File: meow-tutor.el [hunk 2]
```diff
@@ -108,43 +107,42 @@
 
  Note: If you want to move the cursor while in Insert mode,
        you may use the arrow keys instead of exiting and
-       reentering Insert mode.
+       re-entering Insert mode.
  Note: The modeline will display your current mode.
-       Notice that when you press \\[meow-insert], 'NORMAL' changes to 'INSERT'.
```
Comment (VitalyAnkh) [inline meow-tutor.el:113]: The default state indicator for meow is:
```
((normal . "NORMAL")
 (motion . "MOTION")
 (keypad . "KEYPAD")
 (insert . "INSERT")
 (beacon . "BEACON"))
```
We could get the value from the editor itself:
```
"%s %s"
(alist-get 'normal meow-replace-state-name-list) 
(alist-get 'insert meow-replace-state-name-list)
```

What do you think?

This PR improves the quality greatly! Thanks!

Comment (skmd0) [reply meow-tutor.el:113]: Sounds good! My elisp knowledge is not great, I don't think I can do this properly. Can you please do it after I merge the PR?

Comment (VitalyAnkh) [reply meow-tutor.el:113]: No problem! I’m also not familiar with elisp, just learn by doing 😸

Comment (DogLooksGood) [reply meow-tutor.el:113]: ya, we should just use the same name in document. the indicator is just a display. 
we don't have to use it in tutor.

**Comment by skmd0** [comment]
Can we merge this?

**Comment by DogLooksGood** [comment]
please wait a moment, let me check whether something is missed

**Comment by DogLooksGood** [comment]
LGTM, let's merge it!

**Pull request merged**